src/oom/test-oomd-util.c

```diff
@@ -57,7 +57,7 @@ static void test_oomd_cgroup_kill(void) {
 
         /* If we don't have permissions to set xattrs we're likely in a userns or missing capabilities */
         r = cg_set_xattr(SYSTEMD_CGROUP_CONTROLLER, cgroup, "user.oomd_test", "test", 4, 0);
-        if (IN_SET(r, -EPERM, -ENOTSUP))
+        if (ERRNO_IS_PRIVILEGE(r) || ERRNO_IS_NOT_SUPPORTED(r))
                 return (void) log_tests_skipped("Cannot set user xattrs");
```

Comment:
I found this pattern first time: `return (void) log_xxx(...);`
I prefer `log_xxx(...); return;`

Reply:
lol you're right why did I do that it's all over the oomd and PSI unit tests... I'll split the lines in another PR so I don't have to restart the CI again

Reply:
Yeah, let's update it later.

Reply:
Oh, I didn't even know this is possible. It's a pretty cool trick.